$NetBSD: patch-ae,v 1.1 2008/01/29 04:18:45 bjs Exp $

If HAVE_FT_GLYPHSLOT_EMBOLDEN is defined in config.h, then
FT_GlyphSlot_Embolden() is called.  This function's prototype
is in <freetype/ftsynth.h>, so conditionally include it.

Implicit declaration bad.  Prototypes good.  Fixes warning.

--- src/xftglyphs.c.orig	2006-12-07 05:17:19.000000000 -0500
+++ src/xftglyphs.c
@@ -25,6 +25,10 @@
 #include "xftint.h"
 #include <freetype/ftoutln.h>
 
+#if HAVE_FT_GLYPHSLOT_EMBOLDEN
+#include <freetype/ftsynth.h>
+#endif
+
 static const int    filters[3][3] = {
     /* red */
 #if 0
